packages/mock-server/src/api/resolvers.ts

```diff
@@ -20,9 +20,14 @@ import {
 
 const getAvailableQuantity = (itemId: string): number => {
   const stockLines = db.get.stockLines.byItemId(itemId);
-  const sumFn = getSumFn('availableNumberOfPacks');
-  const quantity = stockLines.reduce(sumFn, 0);
-  return quantity;
+  const availableQuantity = stockLines.reduce(
+    (sum, { availableNumberOfPacks, packSize }) => {
+      return sum + availableNumberOfPacks * packSize;
+    },
+    0
+  );
```

Comment:
wasn't sure if this was right, but I think I originally just summed the packs as we were using a pack size of 1.

I thought the items detail modal might need to have the available quantity adjusted by whatever packsize is selected, but unsure

Reply:
oh, yes - that's a good catch.